tests/transforms/test_adjoint.py

```diff
@@ -65,6 +65,18 @@ def my_circuit():
     assert my_circuit()[0] == 1.0
 
 
+def test_wirecut_adjoint():
+    """Check that the adjoint for the WireCut is working"""
+    dev = qml.device("default.qubit", wires=1)
+
+    @qml.qnode(dev)
+    def my_circuit():
+        adjoint(qml.WireCut)(wires=0)
+        return qml.state()
+
+    assert my_circuit()[0] == 1.0

```

Comment:
```suggestion
    assert np.isclose(my_circuit()[0], 1.0)
```
I get paranoid about equality with floats!